bundles/org.openhab.binding.mercedesme/src/main/resources/OH-INF/thing/hvac-channel-types.xml

```diff
@@ -10,14 +10,28 @@
 	</channel-type>
 	<channel-type id="temperature-hvac">
 		<item-type>Number:Temperature</item-type>
-		<label>AC Temperature</label>
+		<label>AC Desired Temperature</label>
```

Comment:
```suggestion
		<label>AC Setpoint</label>
```
(for your consideration)

Reply:
True - changed to `AC Setpoint Temperature`